Refactoring WMS GetFeatureInfo paragraph (#3384)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_ogc/server/services.rst

+---------------+----------+----------------------------------------------+
| BBOX          | No       | :ref:`See GetMap <qgisserver-wms-getmap>`    |
+---------------+----------+----------------------------------------------+
| WIDTH         | Yes      | :ref:`See GetMap <qgisserver-wms-getmap>`    |
+---------------+----------+----------------------------------------------+
| HEIGHT        | Yes      | :ref:`See GetMap <qgisserver-wms-getmap>`    |
+---------------+----------+----------------------------------------------+
| TRANSPARENT   | No       | :ref:`See GetMap <qgisserver-wms-getmap>`    |
+---------------+----------+----------------------------------------------+
| INFO_FORMAT   | No       | Output format                                |
+---------------+----------+----------------------------------------------+
| QUERY_LAYERS  | Yes      | Layers to query                              |
+---------------+----------+----------------------------------------------+
| FEATURE_COUNT | No       | Maximum number of features to return         |
+---------------+----------+----------------------------------------------+

> | X             | No       | X coordinate in pixels of the point to query |

Reviewer: ```suggestion
| X             | No       | Pixel column of the point to query |
```

Reviewer: Should we switch so that I gets this description and X gets "Same as...".

+---------------+----------+----------------------------------------------+
| I             | No       | Same as `X` parameter, but in WMS 1.3.0      |
+---------------+----------+----------------------------------------------+

> | Y             | No       | Y coordinate in pixels of the point to query |

Reviewer: ```suggestion
| Y             | No       | Pixel row of the point to query (0 on top) |
```

Reviewer: Should we switch so that J gets this description and Y gets "Same as..."

+----------------------+----------+------------------------------------------+
| FILTER_GEOM          | No       | Geometry filtering                       |
+----------------------+----------+------------------------------------------+
| WITH_MAPTIP          | No       | Add map tips to the output               |
+----------------------+----------+------------------------------------------+
| WITH_GEOMETRY        | No       | Add geometry to the output               |
+----------------------+----------+------------------------------------------+

|

URL example:

.. code-block:: none

  http://localhost/qgis_server?
  SERVICE=WMS
  &VERSION=1.3.0
  &REQUEST=GetMap
  &MAP=/home/user/project.qgs
  &LAYERS=mylayer1,mylayer2,mylayer3
  &CRS=EPSG:4326
  &WIDTH=400
  &HEIGHT=400
  &INFO_FORMAT=text/xml
  &TRANSPARENT=TRUE
  &QUERY_LAYERS=mylayer1
  &FEATURE_COUNT=3
  &I=250
  &J=250

INFO_FORMAT
^^^^^^^^^^^

This parameter may be used to specify the format of the result. Available
values are:

- ``text/xml``
- ``text/html``
- ``text/plain``
- ``application/vnd.ogc.gml``
- ``application/json``

QUERY_LAYERS
^^^^^^^^^^^^

> This parameter specifies the layers to display on the map. Names have to be

Reviewer 2: "have to be" is not quite right, but I know that words like "should" and "must" should not be used interchangeably. Possible alternatives:

"Names should be separated by commas"
"Names are separated by commas"
"Names should be/are comma-separated"

Reviewer: "Names are separated by commas".  Avoid "should".

separated by a comma.

> In addition, QGIS Server introduced some options to select layers by:

Reviewer 2: ```suggestion
In addition, QGIS Server introduces options to select layers by:
```

> * a short name

Reviewer 2: ```suggestion
* short name
```

> * the layer id

Reviewer 2: ```suggestion
* layer id
```

> You may take a look to the ``LAYERS`` parameter defined in

Reviewer 2: ```suggestion
See the ``LAYERS`` parameter defined in
```

:ref:`See GetMap <qgisserver-wms-getmap>` for more information.

FEATURE_COUNT
^^^^^^^^^^^^^

This parameter specifies the maximum number of features to return.

By default, only 1 feature is returned.

X
^

> This parameter, defined in WMS 1.1.0, allows to specify the X coordinate in

Reviewer 2: ```suggestion
This parameter, defined in WMS 1.1.0, allows you to specify the X coordinate in
```

> pixels of the query point.

Reviewer: Same comment as above.

I
^

> Same parameter than ``X``, but defined in WMS 1.3.0.

Reviewer 2: ```suggestion
Same parameter as ``X``, but defined in WMS 1.3.0.
```

Y
^

> This parameter, defined in WMS 1.1.0, allows to specify the Y coordinate in

Reviewer 2: ```suggestion
This parameter, defined in WMS 1.1.0, allows you to specify the Y coordinate in
```

> pixels of the query point.

Reviewer: Same comments as above.

J
^

> Same parameter than ``Y``, but defined in WMS 1.3.0.

Reviewer 2: ```suggestion
Same parameter as ``Y``, but defined in WMS 1.3.0.
```

FI_POINT_TOLERANCE
^^^^^^^^^^^^^^^^^^

This parameter specifies the tolerance in pixels for point layers.

FI_LINE_TOLERANCE
^^^^^^^^^^^^^^^^^

This parameter specifies the tolerance in pixels for line layers.

FI_POLYGON_TOLERANCE
^^^^^^^^^^^^^^^^^^^^

This parameter specifies the tolerance in pixels for polygon layers.

FILTER_GEOM
^^^^^^^^^^^

This parameter specifies a WKT geometry with which features have to intersect.

WITH_MAPTIP
^^^^^^^^^^^

> This parameter specifies if map tips have to be added to the output.

Reviewer 2: ```suggestion
This parameter specifies whether map tips have to be added to the output.
```

Reviewer: Perhaps change "have to be" to something else: "whether map tips shall be added to the output"?

Available values are (not case sensitive):

- ``TRUE``
- ``FALSE``

WITH_GEOMETRY
^^^^^^^^^^^^^

> This parameter specifies if geometries have to be added to the output.

Reviewer 2: ```suggestion
This parameter specifies whether geometries have to be added to the output.
```

Reviewer: Perhaps change "have to be" to something else: "whether map tips shall be added to the output"?

Reviewer 2: "whether to add map tips to the output", which also avoids use of the passive
